src/test/java/dk/kb/present/transform/XSLTPreservicaToSolrTransformerTest.java

```diff
@@ -28,6 +32,9 @@ String getXSLT() {
 
     @BeforeAll
     public static void beforeMethod() {
+        if (Resolver.getPathFromClasspath("internal_test_files/tvMetadata") == null){
+            log.warn("Internal test files are not present. Unittest 'XSLTPreservicaToSolrTransformerTest' is therefore not run.");
+        }
         Assumptions.assumeTrue(Resolver.getPathFromClasspath("internal_test_files/tvMetadata") != null);
```

Comment:
Alternatively just
```
        Assumptions.assumeTrue(Resolver.getPathFromClasspath("internal_test_files/tvMetadata") != null,
                "Internal test files are not present. Unittest 'XSLTPreservicaSchemaOrgTransformerTest' is therefore not run.");
```
For unknown reasons IntelliJ IDEA eats the `INFO`-level log entry from this but it shows when running `mvn package` from the command line.